### PR TITLE
Improve usability across admin views

### PR DESCRIPTION
**Pull request by jrbotros**

### CODE REVIEW

File: orchestra/models.py [hunk 1]
```diff
@@ -506,6 +507,9 @@ class TimeEntry(BaseModel):
             Server timestamp for timer stop (not null if TimeEntry is
             created using work timer)
     """
+    class Meta:
+        verbose_name_plural = "time entries"
```
Comment (marcua) [inline orchestra/models.py]: single quotes

File: orchestra/admin.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    search_fields = ('workflow__slug', 'slug', 'name', 'description',)
+    list_filter = ('workflow',)
+
+
+def edit_link(instance, text=None):
+    if not instance:
+        return None
+    if text is None:
+        text = str(instance)
+    change_url_name = 'admin:{}_change'.format(instance._meta.db_table)
```
Comment (marcua) [inline orchestra/admin.py]: sanity check---the views are named after the DB tables?  it feels like that breaks encapsulation, but if you've read that this is how it works, then OK

File: orchestra/admin.py [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+from orchestra.models import Worker
+from orchestra.models import WorkerCertification
+from orchestra.models import Workflow
+from orchestra.models import WorkflowVersion
+
+admin.site.site_header = 'Orchestra'
+admin.site.site_title = 'Orchestra'
+admin.site.index_title = 'Orchestra'
+
+
```
Comment (marcua) [inline orchestra/admin.py:22]: sanity check---the number of admin.register reigstrations is the same as it was before this PR?

**Comment by marcua** [comment]
lgtm assuming the two sanity checks pan out

File: orchestra/admin.py (hunk 3 above)
Comment (jrbotros) [inline orchestra/admin.py:22]: ✅

File: orchestra/admin.py (hunk 2 above)
Comment (jrbotros) [inline orchestra/admin.py]: Agreed—will work with other `_meta` props instead.

File: orchestra/models.py (hunk 1 above)
Comment (jrbotros) [inline orchestra/models.py]: ✅

**Pull request merged**